.test-infra/jenkins/Flink.groovy

```diff
@@ -17,8 +17,8 @@
  */
 
 class Flink {
-  private static final String flinkDownloadUrl = 'https://archive.apache.org/dist/flink/flink-1.10.0/flink-1.10.0-bin-scala_2.11.tgz'
-  private static final String hadoopDownloadUrl = 'https://repo.maven.apache.org/maven2/org/apache/flink/flink-shaded-hadoop-2-uber/2.8.3-9.0/flink-shaded-hadoop-2-uber-2.8.3-9.0.jar'
+  private static final String flinkDownloadUrl = 'https://archive.apache.org/dist/flink/flink-1.9.1/flink-1.9.1-bin-scala_2.11.tgz'
```

Comment:
Do you know what the problem was upgrading to Flink 1.10?

Reply:
The CLI of Yarn changed and no longer accepts one of options we supply. Kamil took care of it in #11146 - I'll remove this commit and rebase onto master in a while.